### ISSUE TITLE
SSH config is unsuccessful on RHEL 6.7

### ISSUE THREAD

**Issue by erwangranger** [sassoftware/virk#9]
On testing the viya-3.3 branch on a rhel 6.7 machine, the SSH config errors out with:
```
TASK [virk.preinstall : We restart the SSHD service] *****************************************************************
skipping: [deployTarget]

TASK [virk.preinstall : Clarifying message in case of issue] *********************************************************
fatal: [deployTarget]: FAILED! => {
    "assertion": "sshd_validation_results.rc is defined and sshd_validation_results.rc == 0",
    "changed": false,
    "evaluated_to": false,
    "msg": [
        "It seems that something went wrong with the file /etc/ssh/sshd_config",
        "You should review the MaxStartups line, and correct it",
        "Do NOT restart the sshd service until the command '/sbin/sshd -T' returns a clean output"
    ]
```
On this particular server, /sbin/sshd does not exist. 
Instead, I do have "/usr/sbin/sshd".

We may be able to simply replace "/bin/sshd" with just "sshd", and also make sure that the command works before using it to validate SSH changes.

**Comment by thpang** [comment]
@erwangranger might need to have a section that finds the base tools we need, i.e. sshd and then sets a variable that can be used or we perhaps check the OS version which seems a bit more cumbersome. Thoughts?

**Comment by erwangranger** [comment]
yes, OS checking is cumbersome. I'd say, make it a variable (SSHD_CMD=sshd), then check if that works/exists, if not, give an error and ask the user to override with the actual location. Rinse, repeat. :-)

**Comment by thpang** [comment]
@screnshaw pull down the latest code base for the viya-3.3 branch and make the update @erwangranger asked about changing the application from a full path to the application itself. I feel that because this is an 'sbin' item, the path will not contain what we need and the code will need to know where to go look for this application. Let us know what you find.

**Comment by thpang** [comment]
@screnshaw he is using base-rhel-6.7-sasit image in OpenStack.

**Comment by screnshaw** [comment]
Changes in progress. I re-ran the test script on both OpenStack rhel-6.7 images and the test showed a FAIL on the 'SSH MaxStartups Config' test in both cases. I missed it somehow.

I've updated the pre.ssh_maxstartups_config role in my environment and re-testing on OpenStack.  When complete I will list the OS images I've tested on and push code for review.

**Comment by screnshaw** [comment]
Ran into some issues with testing, but all is complete and successful. Tested on RHEL 6.7, 7, 7.2, 7.3 and 7.4, and Oracle Linux 6.7 and 7.2.

**Comment by thpang** [comment]
This was closed by #14
